edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.xml

```diff
@@ -152,7 +155,22 @@
 
   </target>
 
-  <target name="debug-deploy" depends="jar,get-target-ip,dependencies" description="Deploy the jar and start the program running.">
+  <target name="debug-deploy" depends="jar,get-target-ip,dependencies" description="Deploy the jar and start the program running.">\
```

Comment:
Thanks, no idea how that got there... or how I missed it when reviewing the diff